erpc_python/erpc/transport.py

```diff
@@ -10,6 +10,8 @@
 import struct
 import socket
 import threading
+from _socket import SHUT_RDWR
```

Comment:
Hi, why you don't use value from socket? Not sure where is from _socket

Reply:
Ah, I originally couldn't find this value in `socket`, so I used the private module. This has been fixed now.